content/app/development/ux/pdf/local-development/_index.en.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Local development
+description: How to configure the generation of PDF on local development environment
+weight: 50
+---
+
+1. Set up localtest according to the [description in app-localtest](https://github.com/Altinn/app-localtest)
+2. When you get to step 2 - add the pdf profile to your docker command  
+   This ensures the new pdf service is created.
+    ```shell
+    docker compose --profile pdf --profile localtest up -d --build
+    ```
+3. Add `PdfGenerationSettings` to you _appsettings.Development.json_ file
+    ```json
+    "PdfGeneratorSettings": {
+      "ServiceEndpointUri": "http://local.altinn.cloud/pdfservice/pdf",
+      "AppPdfPageUriTemplate": "http://{hostName}/{appId}/#/instance/{instanceId}?pdf=1",
```

Comment:
Why is `ServiceEndpointUri` set to `local.altinn.cloud`, while `AppPdfPageUriTemplate` uses `{hostName}`. Also, Is there any chance you would provide this as "default values", so that we don't need to fill up appsettings with more boilierplate?

Reply:
Obviously that would mean production pods needs to set correct values for these configurations, but that is one script change, instead of us updating 50 appsettings.json where at least one is getting a subtle bug.